Hunted Mobile/Hunted Mobile/Hunted Mobile/Service/Gps/GpsService.cs

```diff
@@ -23,8 +23,9 @@ public class GpsService {
         /// <summary>
         ///  Handles PositionChanged events from CrossGeolocator.Current by calling the local LocationChanged event property
         /// </summary>
-        private void HandlerLocationChangedEvent(object sender, PositionEventArgs args) {
-            LocationChanged(new Location(args.Position));
+        private void HandlerLocationChangedEvent(object sender, PositionEventArgs args = null) {
+            var location = args != null ? new Location(args.Position) : null;
+            LocationChanged(location);
```

Comment:
```suggestion
            LocationChanged(new Location(args?.Position));
```
Volgens mij is dit hetzelfde